ansible/roles/stack-sunbird/templates/sunbird_player.env

```diff
@@ -113,4 +113,21 @@ reportsListVersion={{reportsListVersion|default('v1')}}
 
 #Release-3.2.7
 sunbird_data_product_service=http://kong:8000/
-discussions_middleware={{ discussionsmw_service_url | default('http://discussionsmw-service:3002') }}
\ No newline at end of file
+discussions_middleware={{ discussionsmw_service_url | default('http://discussionsmw-service:3002') }}
+
+#Release-3.6.0
+sunbird_kong_device_register={{sunbird_kong_device_register|default('false')}}
+sunbird_kong_device_register_token={{sunbird_kong_device_register_token|default('')}}
+
+# Allowed type sentinel/cluster/standalone(default)
+# For sentinel and cluster use below.
+sunbird_redis_type={{sunbird_redis_type|default('standalone')}}
+sunbird_redis_urls={{sunbird_redis_urls|default("redis.dev.svc.cluster.local")}}
+sunbird_redis_port={{sunbird_redis_port|default(26379)}}
+sunbird_redis_password={{sunbird_redis_password|default('none')}}
+
+# For standalone
+# Connect to 127.0.0.1:6380, db 4, using password "authpassword":
+# Ex: redis://:authpassword@127.0.0.1:6380/4
+# Ex: rediss://redis.my-service.com - when using TLS encryption
```

Comment:
please check the password filed here.
isn't it supposed to be rediss://:apssword@redis.my-service.com ?